Formula/ooniprobe.rb

```diff
@@ -166,16 +166,57 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources"
-    system bin/"oonideckgen", "-o", "#{HOMEBREW_PREFIX}/share/ooni/decks/"
-  end
-
   def caveats; <<-EOS.undent
-    Decks are installed to #{HOMEBREW_PREFIX}/share/ooni.
+    Decks are installed to #{HOMEBREW_PREFIX}/share/ooni/decks/.
     EOS
   end
 
+  def plist; <<-EOS.undent
+   <?xml version="1.0" encoding="UTF-8"?>
+   <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+   <plist version="1.0">
+   <dict>
+     <key>Label</key>
+       <string>#{plist_name}</string>
+     <key>Program</key>
+       <string>#{opt_bin}/ooniprobe</string>
+     <key>ProgramArguments</key>
+     <array>
+       <string>-i</string>
+       <string>#{HOMEBREW_PREFIX}/share/ooni/decks/current.deck</string>
```

Comment:
It's a symlink, not an actual directory. For example:
```
~> ls -l /usr/local/share/autogen
/usr/local/share/autogen -> ../Cellar/autogen/5.18.7/share/autogen
```
& `opt_prefix` is a symlink to `/Cellar/formula/current_linked_ver`.

Very few things Homebrew puts in the top-level `HOMEBREW_PREFIX` are _real_ directories, outside of `etc`, `var` and a few special cases here & there elsewhere.